packages/recommendations-js/README.md

```diff
@@ -26,12 +26,49 @@ To get started, you need a container for your results to go in. If you don’t h
 
 Then, inject results into it by calling the `relatedProducts` function and providing the [`container`](#container). It can be a [CSS selector](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors) or an [Element](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement).
 
+### Default view
+
+```js
+/** @jsx h */
+import { h } from 'preact';
+import { relatedProducts } from '@algolia/recommendations-js';
+import algoliasearch from 'algoliasearch';
+
+const appId = 'HYDY1KWTWB';
+const apiKey = '28cf6d38411215e2eef188e635216508';
+const indexName = 'gstar_demo_test';
+
+const searchClient = algoliasearch(appId, apiKey);
+const currentObjectID = 'YOUR_OBJECT_ID';
+
+relatedProducts({
+  container: '#relatedProducts',
+  searchClient,
+  indexName,
+  objectIDs: [currentObjectID],
+  itemComponent({ item }) {
+    return (
+      <pre>
+        <code>{JSON.stringify(item)}</code>
+      </pre>
```

Comment:
can you use it without preact (document.createElement) too, or does that not make sense?

Reply:
That would be either VDOM or HTML strings (see https://github.com/algolia/ui-components/pull/27#issuecomment-856578530).